### PR TITLE
add(modules) widget module

### PR DESCRIPTION
**Pull request by chrisgervang**
For #7946 

#### Background

With the exception of tooltip, the plan is to distribute widgets in their own module. 
#### Change List
- adding widgets module
- documentation

@Pessimistress, should I target master or `9.0-dev`?

### CODE REVIEW

**Review by ibgreen — APPROVED** [review]
Given the ongoing WebGL work - If we are working on an 8.10-alpha track (which makes perfect sense especially for modules like this that don't touch and GPU rendering) I recommend we do so on an `8.10-dev` branch. 

`master` needs to move on to 9.0 soon.

File: docs/api-reference/widgets/overview.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+new FullscreenWidget({});
+```
+
+### Include the Standalone Bundle
+
+```html
+<script src="https://unpkg.com/deck.gl@^8.10.0/dist.min.js"></script>
+<!-- or -->
+<script src="https://unpkg.com/@deck.gl/core@^8.10.0/dist.min.js"></script>
+<script src="https://unpkg.com/@deck.gl/widgets@^8.10.0/dist.min.js"></script>
```
Comment (Pessimistress) [inline docs/api-reference/widgets/overview.md:30]: The next version will be 9.0

**Review by Pessimistress — APPROVED** [review]

**Comment by chrisgervang** [comment]
I'm going to merge this as-is since adding FullscreenWidget will be more discussion and this way other modules can get started in parallel.

File: docs/api-reference/widgets/overview.md (hunk 1 above)
Comment (chrisgervang) [reply docs/api-reference/widgets/overview.md:30]: Opps, missed this. I'll update to 9

**Pull request merged**